sportsdataverse/cfb/cfb_pbp.py

```
@@ -1118,6 +1118,49 @@ def _parse_espn_player_box(boxscore):
+    Columns: ``AirYds`` (all attempts), ``aDOT`` (mean air yards per attempt with
+    data), ``CompAirYds`` (completions only), ``YAC``, and ``AirYdsPct``
+    (``CompAirYds / Yds`` -- share of receiving yards earned in the air).
+    """
+    has_air = pl.col("air_yards").is_not_null()
+    return (
+        pass_box.group_by(["pos_team", key])
+        .agg(
+            _n_air=has_air.sum(),
+            AirYds=pl.col("air_yards").sum(),
+            aDOT=pl.col("air_yards").mean(),
+            CompAirYds=pl.when(pl.col("completion") == True).then(pl.col("air_yards")).otherwise(None).sum(),
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)